includes/Modules/Analytics/Provisioning.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Analytics\Provisioning
+ *
+ * @package   Google\Site_Kit\Modules\Analytics
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\Analytics;
+
+use Google\Site_Kit\Core\Modules\Module_Settings;
+use Google\Site_Kit\Core\Storage\Setting_With_Legacy_Keys_Trait;
+use  Google\Site_Kit\Modules\Analytics\AccountTicket;
+/**
+ * Class for Analytics provisioning.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Provisioning extends \Google\Site_Kit_Dependencies\Google_Service_Resource {
```

Comment:
Same here, maybe `Proxy_Provisioning`? Also better add a `use` statement for that service class rather than having the full name here.

Reply:
changed, moved to use